gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -74,6 +74,36 @@ to generate set of instructions that includes a command block with the registrat
 // tag::token-k8s-heading[]
 == Register and Run with a Token in a Kubernetes Cluster
 // end::token-k8s-heading[]
+// tag::app-openshift-heading[]
+== Register and Run with a Connected App on an OpenShift Cluster
+// end::app-openshift-heading[]
+// tag::user-openshift-heading[]
+== Register and Run with a Username and Password on an OpenShift Cluster
+// end::user-openshift-heading[]
+// tag::token-openshift-heading[]
+== Register and Run with a Token on an OpenShift Cluster
+// end::token-openshift-heading[]
+
```

Comment:
this is just a note for the openshift sections to say that the command-line procedures are the same as for a k8 cluster deployment